frontend/ui/data/creoptions.lua

```diff
@@ -178,6 +178,21 @@ Note that your selected font size is not affected by this setting.]]),
 - off: no hinting.
 - native: use the font internal hinting instructions.
 - auto: use FreeType's hinting algorithm, ignoring font instructions.]]),
+            },
+            {
+                name = "font_kerning",
+                name_text = S.FONT_KERNING,
+                toggle = {S.OFF, S.FREETYPE, S.HARFBUZZ},
+                values = {0, 1, 2},
+                default_value = 1,
+                args = {0, 1, 2},
+                event = "SetFontKerning",
+                name_text_hold_callback = optionsutil.showValues,
+                help_text = _([[Font kerning is the process of adjusting the spacing between individual letter forms, to achieve a visually pleasing result.
+
+- off: no kerning.
+- freetype: use FreeType's kerning implementation (no ligature).
+- harfbuzz: use Harfbuzz's kerning implementationg (slower but may support ligatures with some fonts).]]),
```

Comment:
We should probably mention something about RTL here as well (Hebrew, Arabic, Persian, etc.)

Reply:
But it doesn't magically make RTL work ! :) It's really incomplete. I mean like:
tuB ti t'nseod yllacigim ekam LTR krow ! ): tI's yllaer etelpmocni. ylleaR!

Or rather, it would help reading words, but the words themselves would be still LTR, like:
like:, LTR still be would themselves words the but, words reading help would it, rather Or.

Reply:
True, but without Harfbuzz things like RTL Unicode combining diacritics "should" attach to the wrong characters and the like.

Just flipping word order sounds like a fairly simple proposition in theory. All that proper Unicode handling — nope, we'd better let some Persian deal with that! ;-)

Reply:
Flipping word order sounds simple indeed. But then, there's all the crengine mapping to the DOM text nodes that fails (finding word boundaries for highlight, searching, clicking on links...) So there's quite a bit of thinking on that side to do.
Which could be interesting. But when you don't read any RTL language, it's some blind fun... :)